src/app/table/table.component.html

```diff
@@ -117,8 +118,20 @@
 </ng-template>
 
 <ng-template #nesting let-item>
-  <button type="button" class="button is-small is-rounded is-white" (click)="onToggleNested(item)">
+  <button type="button" class="button is-small is-rounded is-white" (click)="onToggleNesting(item)">
     <ng-container *ngIf="isNestedOpened(item)">▲</ng-container>
     <ng-container *ngIf="!isNestedOpened(item)">▼</ng-container>
   </button>
+</ng-template>
+
+<ng-template #sorting let-column>
+  <span (click)="onToggleSorting(column)" class="is-clickable" style="border-bottom: 1px dashed">
+    {{column.label}}
+    <span class="is-pulled-right">
+      <ng-container *ngIf="getSrotingOrder(column) === TableConfigSortingOrders.Desc">▲</ng-container>
```

Comment:
Will take a look and will back to you...

Reply:
Super useful. Thanks.
What about if
```html
*ngIf="getSortingOrder(column)"
```
will become 
```html
*ngIf="selectedItems.get(column)"
```
where `selectedItems` is `Map()` ? It will not reduces performance, right? I don't know how to verify that case. Logically it should't